Problem1.java

```diff
@@ -0,0 +1,40 @@
+// Time Complexity : O(N)
+// Space Complexity : O(N)
+// Did this code successfully run on Leetcode : Y
+// Any problem you faced while coding this : N
+
```

Comment:
Good Job! Keep up the good work. Please add some comments explaining your approach.